package.json

```diff
@@ -4,6 +4,7 @@
   "description": "enhance your progress bars with minimal effort!",
   "main": "dist/ep.js",
   "scripts": {
+    "prepublish": "make dist",
```

Comment:
If I understand correctly, this is why `dist` has been ignored? It's assumed that a user will pull the package in via `npm` and therefore creating the `dist` output `prepublish` makes it available to those who pull in via `npm` etc.?

Reply:
Yep, this script make npm build the package before it is published to the npm central repository, therefore after npm users installing the package, the `dist` will just be there. In this way, we can guarantee that the build is always newest and relax us from manually building.